loader/CMakeLists.txt

```diff
@@ -409,6 +409,13 @@ if(PKG_CONFIG_FOUND)
     else()
         set(CMAKE_INSTALL_FULL_LIBDIR_PC ${CMAKE_INSTALL_FULL_LIBDIR})
     endif ()
+    if ("${CMAKE_INSTALL_PREFIX}" STREQUAL "")
+        set(CMAKE_INSTALL_REL_LIBDIR_PC ${CMAKE_INSTALL_FULL_LIBDIR_PC})
+        set(CMAKE_INSTALL_REL_INCLUDEDIR_PC ${CMAKE_INSTALL_FULL_INCLUDEDIR})
+    else()
+        file(RELATIVE_PATH CMAKE_INSTALL_REL_LIBDIR_PC ${CMAKE_INSTALL_PREFIX} ${CMAKE_INSTALL_FULL_LIBDIR_PC})
+        file(RELATIVE_PATH CMAKE_INSTALL_REL_INCLUDEDIR_PC ${CMAKE_INSTALL_PREFIX} ${CMAKE_INSTALL_FULL_INCLUDEDIR})
+    endif()
```

Comment:
This change is to support the following configuration.

```
cmake \
        -DCMAKE_INSTALL_PREFIX=/usr \
        -DCMAKE_INSTALL_LIBDIR=/usr/lib64 \
        -DCMAKE_INSTALL_INCLUDEDIR=/usr/include
```

I've never seen this before. AFAIK users are typically only supposed to set `CMAKE_INSTALL_PREFIX`. Granted this may not be a typical case.

Also the generation of `vulkan.pc` shouldn't happen at project generation time. It should happen at installation time. It will require the usage of `install(SCRIPT ...)` or `install(CODE ...)` to do this properly.

Reply:
> Therefore please don't expect this changes in this PR. If that would be prerequisite feel free to reject it.

I don't expect it to be a prereq. I just wanted to highlight what the most correct thing to do would be for future reference after looking into this.